src/docs/common_tasks/multiple_jvm_versions.md

```diff
@@ -0,0 +1,53 @@
+
+## Problem
+
+You want to compile and/or run different test or binary targets using different JVMs.
+
+Or, you need to specify command-line options globally for the JVM on a per-JVM version basis. For
+example, using `--add-opens` flags, but only when running on JVMs that support it.
+
+## Solution
+
+Declare JVM platforms using the `jvm-platform` configuration section to define the set of available
+platforms.
+
+On targets, add the kwarg `runtime_platform` set to the name of one of the platforms defined in
```

Comment:
```suggestion
On targets, add the named argument `runtime_platform` set to the name of one of the platforms defined in
```

`kwarg` is a Pythonic word, and a lot of users don't use Python that much.

Reply:
Thanks. I think that would be a good thing to have in a doc guideline doc somewhere.